tests/layer_tests/pytorch_tests/test_gather.py

```diff
@@ -6,10 +6,10 @@
 from pytorch_layer_test_class import PytorchLayerTest, skip_if_export
 
 
-class TestGelu(PytorchLayerTest):
+class TestGather(PytorchLayerTest):
     def _prepare_input(self, m, n, max_val, out=False):
         import numpy as np
-        index = np.random.randint(0, max_val, (m, n))
+        index = np.random.randint(0, max_val, (m, n)).astype(np.int64)
```

Comment:
as I know int64 is deprecated and removed in some numpy versions
```suggestion
        index = np.random.randint(0, max_val, (m, n)).astype(int)
```